cmd/askd/handlers/form_submission.go

```diff
@@ -253,3 +307,25 @@ func (formSubmissionHandle) RemoveFlag(c *app.Context) error {
 
 	return nil
 }
+
+// encodeCSV gets all the submissions and encode them into a CSV
+func encodeCSV(s []submission.Submission) ([]byte, error) {
+
+	var buf bytes.Buffer
+	w := csv.NewWriter(&buf)
+	if err := w.Write(s[0].GetQuestions()); err != nil {
```

Comment:
Only concern for this is what the CSV would look like if the questions changed halfway through the submission collection, which is why with each submission we also grab the question. This may cause a more complex looking header detection function though :(